localstack/services/dynamodb/provider.py

```diff
@@ -1197,6 +1205,56 @@ def describe_kinesis_streaming_destination(
             KinesisDataStreamDestinations=stream_destinations, TableName=table_name
         )
 
+    #
+    # Continuous Backups
+    #
+
+    def describe_continuous_backups(
+        self, context: RequestContext, table_name: TableName
+    ) -> DescribeContinuousBackupsOutput:
+        self.get_global_table_region(context, table_name)
+        store = get_store(context.account_id, context.region)
+        continuous_backup_description = (
+            store.table_properties.get(table_name, {}).get("ContinuousBackupsDescription")
+        ) or ContinuousBackupsDescription(
+            ContinuousBackupsStatus=ContinuousBackupsStatus.ENABLED,
+            PointInTimeRecoveryDescription=PointInTimeRecoveryDescription(
+                PointInTimeRecoveryStatus=PointInTimeRecoveryStatus.DISABLED
+            ),
+        )
+
+        return DescribeContinuousBackupsOutput(
+            ContinuousBackupsDescription=continuous_backup_description
+        )
+
+    def update_continuous_backups(
+        self,
+        context: RequestContext,
+        table_name: TableName,
+        point_in_time_recovery_specification: PointInTimeRecoverySpecification,
+    ) -> UpdateContinuousBackupsOutput:
+        self.get_global_table_region(context, table_name)
+
+        store = get_store(context.account_id, context.region)
+        pit_recovery_status = (
+            PointInTimeRecoveryStatus.ENABLED
+            if point_in_time_recovery_specification["PointInTimeRecoveryEnabled"]
+            else PointInTimeRecoveryStatus.DISABLED
+        )
+        continuous_backup_description = ContinuousBackupsDescription(
+            ContinuousBackupsStatus=ContinuousBackupsStatus.ENABLED,
+            PointInTimeRecoveryDescription=PointInTimeRecoveryDescription(
+                PointInTimeRecoveryStatus=pit_recovery_status
+            ),
+        )
+        store.table_properties[table_name] = {
```

Comment:
This would potentially overwrite other table properties - I believe this should be something like:

```
        table_props = store.table_properties.setdefault(table_name, {})
        table_props["ContinuousBackupsDescription"] = continuous_backup_description
```

Reply:
Good catch! You are absolutely right 🙏